ghost/admin/app/utils/analytics.js

```diff
@@ -34,6 +34,10 @@ export function trackEvent(eventName, props = {}) {
         (window.plausible.q = window.plausible.q || []).push(arguments);
     };
     window.plausible(eventName, {props: props});
+
+    if (window.posthog) {
```

Comment:
One thing I've just learned this week — apparently `window.posthog` [can _exist_ but not be fully loaded](https://linear.app/tryghost/issue/PA-36/windowposthogget-distinct-id-fails-even-when-windowposthog-exists). As a quick fix for the time being, I've started using `window.posthog.__loaded` to confirm it's fully loaded before calling any functions on it — we should probably do that here too!